contract/vm.go

```diff
@@ -600,27 +607,31 @@ func (ce *executor) call(instLimit C.int, target *LState) C.int {
 	}
 	ce.processArgs()
 	if ce.err != nil {
-		ctrLgr.Debug().Err(ce.err).Str("contract",
-			types.EncodeAddress(ce.ctx.curContract.contractId)).Msg("invalid argument")
+		ctrLgr.Debug().Err(ce.err).Stringer("contract",
+			types.LogAddr(ce.ctx.curContract.contractId)).Msg("invalid argument")
```

Comment:
I don't understand why this change
What is the difference int the log output?

Reply:
This is one of the log optimization. The former calls the EncodeAddress function regardless of the log level, but the latter does not call the encode function if the log level is lower than Debug. The outputs are same.